integration_test/EmitVerilog/instance-choice.mlir

```diff
@@ -0,0 +1,54 @@
+// REQUIRES: verilator
+// RUN: circt-opt %s -export-split-verilog='dir-name=%t.dir' -o /dev/null
```

Comment:
Why aren't you using circt-rtl-sim?

Reply:
The tool makes too many assumptions about the source file list. Here we want to pass in a bunch of addition `-F` files and options in order since we are testing the ABI against a specific tool.